docs/data-sources/kafka_cluster_v2_instance.md

```diff
@@ -119,7 +119,7 @@ Adds the specified version of Kafka REST Proxy to this Kafka cluster.<br>
 <br>Integrates the REST proxy with the Schema registry attached to this cluster. Requires 'integrateRestProxyWithSchemaRegistry' to be 'true'.<br><br>
 *___version___*<br>
 <ins>Type</ins>: string, read-only<br>
-<ins>Constraints</ins>: pattern: `[0-9]+\.[0-9]+\.[0-9]+`<br><br>Adds the specified version of Kafka REST Proxy to the Kafka cluster. Available versions: <ul> <li>`5.0.4`</li> <li>`5.0.0`</li> </ul><br><br>
+<ins>Constraints</ins>: pattern: `[0-9]+\.[0-9]+\.[0-9]+`<br><br>Adds the specified version of Kafka REST Proxy to the Kafka cluster. Available versions: <ul> </ul><br><br>
```

Comment:
is this add-on moving off GA?
there's no version available

Reply:
yes true, no GA for this add-on from now. There are karapace addons replacing this.